pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -256,9 +256,9 @@ def test_assert_no_banners(self):
         self.pyscript_run(
             """
             <py-script>
-                import pyscript
-                pyscript.showWarning("hello")
-                pyscript.showWarning("world")
+                from _pyscript_js import showWarning
+                showWarning("hello")
+                showWarning("world")
```

Comment:
I think we still want to expose `pyscript.showWarning` and make it generally available to users